### PR TITLE
add 404Page-container

### LINKED ISSUE #77: Frontend: 404Page
When I try to access a page/URL/route that doesn't exist I want to be able to see the 404 page so I can be reminded that the page doesn't exist.

Create the page below using the existing components.

Screenshot 2020-11-26 at 17 28 24

Be aware that the page should be named and work accordingly to these issues #61 and #60 

Note: Create the components with the appropriate style. Please check the wireframes https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by daxiousm**
# Description

When I try to access a page/URL/route that doesn't exist I want to be able to see the 404 page so I can be reminded that the page doesn't exist.

Fixes #77 (issue)

# How to test
after git pull do 'npm run storybook'

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/footer/footer.styles.css [hunk 1]
```diff
@@ -1,8 +1,9 @@
 footer {
   background-color: #f8f8f8;
-  position: fixed;
   width: 100%;
```
Comment (tpetrina) [inline src/client/components/footer/footer.styles.css:3]: Why did you remove `position: fixed`?

File: src/client/containers/404Page/404Page.styles.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.page404-text {
+  text-align: center;
+  font-weight: 50px;
+  padding: 10rem 0;
+  color: #bdbdbd;
+}
+
+.text-404 {
+  font-size: 100px !important;
```
Comment (tpetrina) [inline src/client/containers/404Page/404Page.styles.css:9]: Why the use of `!important`? Please remove it.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Looks almost good, but the storybook is broken. Please fix.

File: src/client/containers/404Page/404Page.styles.css (hunk 2 above)
Comment (daxiousm) [reply src/client/containers/404Page/404Page.styles.css:9]: removed

File: src/client/components/footer/footer.styles.css (hunk 1 above)
Comment (daxiousm) [reply src/client/components/footer/footer.styles.css:3]: to scroll up and down with the page...but i put it back
